docs/timeout-activities-and-workflows.md

```diff
@@ -0,0 +1,119 @@
+---
+id: timeout-activities-and-workflows
+title: How to timeout Activities and Workflows
+sidebar_label: Timeout Activities and Workflows
+---
+
+Temporal does not impose limits on the duration of Activities or Workflows by default, it is designed to enable these to run idefinitely if desired. Instead, timeout options are available to set limits on the time allowed for certain actions to occur.
+
+## Activity timeouts
+
+Activity timeouts provide additional control over the way your Workflows execute Activities.
+
+### Timeout policy options
+
+The following fields values can be customized within the `ActivityOptions`:
+
+| Field | Java Type | Go Type | Description | Default |
+|-------|-----------|---------|-------------|---------|
+| `ScheduleToStart` | Duration | time.Time | Maximum time a Workflow should wait for a Worker to start executing the Activity. This is commonly used for checking if all workers are down or are not able to keep up with current request rate. | TODO |
```

Comment:
I would say that in some rare cases it is useful for a workflow to react when workers are down. One such use case is task routing to specific hosts. In this case, if the host is down it is not expected that it is going to come back (at least in K8s), so setting this timeout to a low value makes sense. But in the majority of situations, this timeout should not be set as it is expected that worker will come back and pick up a task.